### PR TITLE
New presence - Mugzone

### PR DESCRIPTION
**Pull request by Himori-0**
I hope this is enough proof
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image
image

### CODE REVIEW

**Comment by EncryptedDev** [comment]
Why is your indentation so inconsistent?

File: websites/M/Mugzone/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "service": "Mugzone",
+    "author": {
+      "name": "Himori",
+      "id": "350602512044261386"
+    },
+    "category": "games",
+    "logo": "https://i.imgur.com/pIJ3aTA.png",
+    "thumbnail": "https://i.imgur.com/JkhV3Q0.png",
```
Comment (Bas950) [inline websites/M/Mugzone/dist/metadata.json:10]: logo  must be at least 512x512

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/Mugzone/dist/metadata.json (hunk 1 above)
Comment (Himori-0) [reply websites/M/Mugzone/dist/metadata.json]: I updated the logo

**Comment by Himori-0** [comment]
> Why is your indentation so inconsistent?

sorry i'm still not very good at coding

**Comment by EncryptedDev** [comment]
> > Why is your indentation so inconsistent?
> 
> 
> 
> sorry i'm still not very good at coding

Use prettier

File: websites/M/Mugzone/dist/metadata.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    ],
+    "settings": [
+    {
+        "id": "privacy",
+        "title": "Keep me hidden",
+        "icon": "fas fa-eye-slash",
+        "value": false
+        
+    }
+    ],
```
Comment (Bas950) [inline websites/M/Mugzone/dist/metadata.json:26]: please fix the empty row

File: websites/M/Mugzone/presence.ts [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+presence.on("UpdateData", async () => {
+const privacy = await presence.getSetting("privacy"),
+  presenceData: PresenceData = {
+    largeImageKey: "malody"
+  };
+if (privacy) {
+  presence.setTrayTitle();
+  presence.setActivity();
+} else {
```
Comment (Bas950) [inline websites/M/Mugzone/presence.ts:14]: You could show text like browsing instead of nothing at all...

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/Mugzone/dist/metadata.json (hunk 2 above)
Comment (Himori-0) [reply websites/M/Mugzone/dist/metadata.json]: fixed!

File: websites/M/Mugzone/presence.ts (hunk 3 above)
Comment (Himori-0) [reply websites/M/Mugzone/presence.ts]: I did that now!Thanks for the suggestion.

**Review by ririxidev — APPROVED** [review]
ok i guess

**Review by Bas950 — APPROVED** [review]
LGTM

**Pull request merged**